pkg/metallb/frr_test.go

```diff
@@ -0,0 +1,348 @@
+package metallb
+
+import (
+	"testing"
+	"time"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/schemes/metallb/frrtypes"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	frrConfigurationGVK = schema.GroupVersionKind{
+		Group:   APIGroup,
+		Version: APIVersion,
+		Kind:    FrrConfigurationKind,
+	}
+	defaultFrrConfigurationName   = "frrconfiguration"
+	defaultFrrConfigurationNsName = "test-namespace"
+)
+
+func TestNewFrrConfigurationBuilder(t *testing.T) {
```

Comment:
The tests should be for all the funs. Please add tests for WithBGPNeighbor , WithBGPRouter

Reply:
added